docs/_config.yml

```diff
@@ -0,0 +1,59 @@
+# Additionally influences open graph metadata as part of jekyll-seo-tag
+title: 'Metasploit Documentation | Penetration Testing Software, Pen Testing Security'
+description: View Metasploit Framework Documentation
+searchTitle: Metasploit Documentation
+baseurl: '/metasploit-framework'
+url: ''
+
+permalink: pretty
+
+heading_anchors: true
```

Comment:
```suggestion
# Just-The-Docs configuration settings
heading_anchors: true
```

Will make it easier to read later on and grouping these settings by plugin will make it easier for people to look up and find which setting relates to what plugin, particularly given we have multiple plugin setting adjustments here.